projects/aca-playwright-shared/src/api/shared-links-api.ts

```diff
@@ -52,4 +51,22 @@ export class SharedLinksApi extends ApiClientFactory {
       return null;
     }
   }
+
+  private async getFavorites(userName: string) {
+    try {
+      return await this.apiService.favorites.listFavorites(userName);
+    } catch (error) {
+      logger.error(`\n--- ${error} error :`);
```

Comment:
Could you maybe add some indicator of where error was thrown into the error message? Something like `Error while fetching favourites ${error} :`

Reply:
added error description.